### ISSUE TITLE
Retrieve label inside tracker URL

### ISSUE THREAD

**Issue by Perick95** [adjust/android_sdk#458]
Hi!

I created a custom tracker URL from the adjust dashboard with an additional [label parameter](https://help.adjust.com/en/article/advanced-tracker-setup#label). The tracker URL works fine and I'm able to see all the data about installation ecc, inside the dashboard.
What i want to know is if it's possibile to retrieve the label parameter from the URL on first install inside my application. 

So basically if the installation comes from that specific URL i need to retrieve my custom label, if is an organic install or comes from some others url I do nothing.

Thanks in advance!

**Comment by uerceg** [comment]
Hi @Perick95 

This should be obtainable inside of SDK's [attribution callback](https://github.com/adjust/android_sdk#af-attribution-callback). Once you have set all parameters of the campaign, it should be accessible inside of `AdjustAttribution` object which gets delivered into attribution callback. Field is called [clickLabel](https://github.com/adjust/android_sdk/blob/master/Adjust/sdk-core/src/main/java/com/adjust/sdk/AdjustAttribution.java#L34).

Let us know if you managed to get it in there.

Cheers

**Comment by Yennefer** [comment]
Hi @uerceg !

I'm actually having the same problem with `campaign` and `label` parameters.
I'm trying to implement the deferred deep linking scenario.
I've generated a link with `campaign` and `label` parameters that looks like this: `https://app.adjust.com/7rg6bpb?deep_link=my-app%3A%2F%2Freg&campaign=someCampaign&label=someLabel`
I have set up `OnAttributionChangedListener` and implemented `onAttributionChanged` callback according to the example in the documentation.
This link is getting opened in Google Play, I download and install my app, but I only get several attributes in `AdjustAttribution` class, such as `trackerToken`, `trackerName`, `network`, and `adid`. I'm not getting a `campaign` nor a `label` attributes that I've set up in my deep link. 

Can you please help me to identify what can the problem with that be?
